packages/diagrams/frontend/sirius-components-diagrams/src/renderer/palette/DiagramPalette.tsx

```diff
@@ -36,7 +36,13 @@ export const DiagramPalette = memo(({ diagramElementId }: DiagramPaletteProps) =
 
   return isOpened && x && y ? (
     <PalettePortal>
-      <Palette x={x} y={y} diagramElementId={diagramElementId} onDirectEditClick={() => {}} />
+      <Palette
+        x={x}
+        y={y}
+        diagramElementId={diagramElementId}
+        diagramElementTargetObjectId={diagramTargetObjectId}
```

Comment:
I don't understand why all this code has to be modified to give access to the `targetObjectId`...

Reply:
it is used to pass the information about the selected semantic element. Without this information, I can't retrieve it directly from org.eclipse.sirius.components.selection.graphql.datafetchers.subscription.SubscriptionSelectionEventDataFetcher

Reply:
done